ccm_web/server/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,37 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { INestApplication } from '@nestjs/common'
+import request from 'supertest'
+import { AppModule } from './../src/app.module'
+
+describe('BaseController (e2e)', () => {
+  let app: INestApplication
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule]
+    }).compile()
+
+    app = moduleFixture.createNestApplication()
+    await app.init()
+  })
+
+  it('/api/globals (GET)', async () => {
+    return await request(app.getHttpServer())
+      .get('/api/globals')
+      .expect(401)
+      .expect(
+        {
+          status: 401,
+          error: 'Unauthorized',
```

Comment:
This is kind of a silly test, but I wanted to demonstrate the use of these tools, and testing whether the LTI middleware is blocking unauthenticated users has some benefit. I (or someone) probably needs to do more research on how we'd actually be able to make authenticated requests.